daemon/cluster/noderunner.go

```diff
@@ -163,6 +165,46 @@ func (n *nodeRunner) handleControlSocketChange(ctx context.Context, node *swarmn
 	}
 }
 
+func (n *nodeRunner) watchClusterEvents(ctx context.Context, conn *grpc.ClientConn) {
+	if n.cluster == nil {
+		return
+	}
+	client := swarmapi.NewStoreClient(conn)
+	watch, err := client.Watch(ctx, &swarmapi.WatchRequest{
+		Entries: []*swarmapi.WatchRequest_WatchEntry{
+			{
+				Kind:   "node",
```

Comment:
We don't want to get `task` change notifications. They are too many and are implementation details that we don't user rely on that.